department-of-veterans-affairs/caseflow
# Div/appeals 58967

Resolves [Add Restriction to Modal to not allow a future date to be selected](https://jira.devops.va.gov/browse/APPEALS-58967)

# Description
Fix for feature date restriction when adding a response letters

## Acceptance Criteria

## Testing Plan

1. Log into caseflow as INBOUND_TEAM_SUPERUSER1
2. Navigate to the Correspondence cases page
3. Click on any appeal on the Completed or Pending tabs that will take you to the Correspondence details page.
4. On the Correspondence details page, click on the Response letters tab
5. Click the +Add letter, a Add response letter modal appears
6. Click on the Date sent field and see that the feature dates are not selectable


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---
<img width="689" alt="Screenshot 2024-09-20 at 11 49 33 AM" src="https://github.com/user-attachments/assets/067fc59a-b1bd-446a-a493-1f34f9327d98">
<img width="596" alt="Screenshot 2024-09-20 at 11 49 55 AM" src="https://github.com/user-attachments/assets/0b25e94c-9786-45c1-9b93-57039543576b">

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
